pkg/resources/grant_helpers.go

```diff
@@ -187,6 +187,8 @@ func readGenericGrant(
 			privileges.addString(grant.Privilege)
 			// Reassign set back
 			sharePrivileges[granteeNameStrippedAccount] = privileges
+		default:
```

Comment:
How the default case fixes the issue ? Right now, the program behaviour is the same except it logs on unknown case in switch

Reply:
actually this was already fixed in https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2037/files
released as part of 0.70.1.

I guess this PR is not needed. it does make it more explicit though

Reply:
Hmm, in the pr you've linked there's also only change with just logging which shouldn't break / fix anything, right ? I don't understand why this or linked pr is fixing anything